src/scraper/items.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+
+# Define here the models for your scraped items
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ScraperItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
It's generated, but scrapy requires it to be in this directory in order for the spiders to work - it only allows a small subset of commands when it's not in any scrapy project.

Some further information on this: 
https://docs.scrapy.org/en/latest/topics/commands.html#default-structure-of-scrapy-projects

Also, here's an example project from scrapy itself which also contains the autogenerated files: 
https://github.com/scrapy/quotesbot